src/sysext/sysext.c

```diff
@@ -169,6 +262,17 @@ static int unmerge_hierarchy(const char *p) {
 
 static int unmerge(void) {
         int r, ret = 0;
+        bool need_to_reload = false;
+
+        if (!arg_no_reload) {
+                int reload;
```

Comment:
don't define a new veriable here, just use `r`, which is where we usually write funciton return values to